packages/asset-buyer/docs/reference.mdx

```diff
@@ -0,0 +1,1300 @@
+
+## Index
+
+### Properties
+
+* [makerAssetData](#makerassetdata)
+* [takerAssetData](#takerassetdata)
+
+## Properties
+
+###  makerAssetData
+
+• **makerAssetData**: *string*
+
+*Defined in [types.ts:10](https://github.com/0xProject/0x-monorepo/blob/b834b7060/packages/asset-buyer/src/types.ts#L10)*
+
+___
+
+###  takerAssetData
+
+• **takerAssetData**: *string*
+
+*Defined in [types.ts:11](https://github.com/0xProject/0x-monorepo/blob/b834b7060/packages/asset-buyer/src/types.ts#L11)*
+
+<hr />
+
+> # Interface: OrderProviderResponse
+
+orders: An array of orders with optional remaining fillable makerAsset amounts. See type for more info.
+
+## Hierarchy
+
+* **OrderProviderResponse**
+
+## Index
+
+### Properties
+
+* [orders](#orders)
+
+## Properties
+
+###  orders
+
+• **orders**: *[SignedOrderWithRemainingFillableMakerAssetAmount](#interface-signedorderwithremainingfillablemakerassetamount)[]*
+
+*Defined in [types.ts:18](https://github.com/0xProject/0x-monorepo/blob/b834b7060/packages/asset-buyer/src/types.ts#L18)*
+
+<hr />
+
+> # Interface: OrdersAndFillableAmounts
+
+orders: An array of signed orders
+remainingFillableMakerAssetAmounts: A list of fillable amounts for the signed orders. The index of an item in the array associates the amount with the corresponding order.
+
+## Hierarchy
+
+* **OrdersAndFillableAmounts**
+
+## Index
+
+### Properties
+
+* [orders](#orders)
+* [remainingFillableMakerAssetAmounts](#remainingfillablemakerassetamounts)
+
+## Properties
+
+###  orders
+
+• **orders**: *`SignedOrder`[]*
+
+*Defined in [types.ts:132](https://github.com/0xProject/0x-monorepo/blob/b834b7060/packages/asset-buyer/src/types.ts#L132)*
+
+___
+
+###  remainingFillableMakerAssetAmounts
+
+• **remainingFillableMakerAssetAmounts**: *`BigNumber`[]*
+
+*Defined in [types.ts:133](https://github.com/0xProject/0x-monorepo/blob/b834b7060/packages/asset-buyer/src/types.ts#L133)*
+
+<hr />
+
+> # Interface: SignedOrderWithRemainingFillableMakerAssetAmount
+
+A normal SignedOrder with one extra optional property `remainingFillableMakerAssetAmount`
+remainingFillableMakerAssetAmount: The amount of the makerAsset that is available to be filled
+
+## Hierarchy
+
+* `SignedOrder`
+
+  * **SignedOrderWithRemainingFillableMakerAssetAmount**
+
+## Index
+
+### Properties
+
+* [exchangeAddress](#exchangeaddress)
+* [expirationTimeSeconds](#expirationtimeseconds)
+* [feeRecipientAddress](#feerecipientaddress)
+* [makerAddress](#makeraddress)
+* [makerAssetAmount](#makerassetamount)
+* [makerAssetData](#makerassetdata)
+* [makerFee](#makerfee)
+* [remainingFillableMakerAssetAmount](#optional-remainingfillablemakerassetamount)
+* [salt](#salt)
+* [senderAddress](#senderaddress)
+* [signature](#signature)
+* [takerAddress](#takeraddress)
+* [takerAssetAmount](#takerassetamount)
+* [takerAssetData](#takerassetdata)
+* [takerFee](#takerfee)
+
+## Properties
+
+###  exchangeAddress
+
+• **exchangeAddress**: *string*
+
+*Inherited from void*
```

Comment:
Hmm do we need these `Inherited from void` lines? They are everywhere, any way to get rid of them?